Intersect.Server/Entities/BankInterface.cs

```diff
@@ -109,6 +109,12 @@ public bool TryDepositItem(int slot, int amount, bool sendUpdate = true)
                 {
                     lock (mLock)
                     {
+                        if (!itemBase.CanBank)
+                        {
+                            PacketSender.SendChatMsg(mPlayer, Strings.Items.nobank, ChatMessageType.Bank, CustomColors.Items.Bound);
+                            return false;
+                        }
+
```

Comment:
Can we move this outside of the lock above? No point in taking the time to acquire a lock if we are just going to return out of the function.

Reply:
Fair, just changed it.